db/README.md

```diff
@@ -0,0 +1,106 @@
+# PostgreSQL Database Migration Steps
+
```

Comment:
Thank you so much Catherine for the awesome readme that developers can follow!!
Hope it is ok for some suggestion (it is very good already, just to be clear for me)

* Can we have the title as "major version upgrade" not "migration"?
* If it is ok, can we also put the "note" section on top to standout to developer's eyes?

Some discussion (if I understand correct)
Maybe pre-steps also needs:
* Create a pr branch before upgrading, update the db/openshift.deployment.yml for DB_VERSION to next version (etc. 18)
* Check out the pr branch in local and use the branch terminal for execution steps.

Reply:
Thanks Ian! Updated, feel free to check again!